config/config.go

```diff
@@ -25,8 +25,7 @@ import (
 
 	autonat "github.com/libp2p/go-libp2p-autonat"
 	blankhost "github.com/libp2p/go-libp2p-blankhost"
-	circuit "github.com/libp2p/go-libp2p-circuit"
-	discovery "github.com/libp2p/go-libp2p-discovery"
+	circuitv2client "github.com/libp2p/go-libp2p-circuit/v2/client"
```

Comment:
that's a huge name; just `circuitv2` will do.

Reply:
Even just `circuit` will do in fact.

Reply:
done.